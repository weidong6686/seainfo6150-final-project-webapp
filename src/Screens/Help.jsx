import React from 'react';
import './css/About.css';
const Help = (props) => {
    return (
        <div className="content">
            <h2 className="title">Hot Questions</h2>
            <h3>[Return Refund] How do I return my wrong/damaged item once the return request has been confirmed?</h3>
            <p>Once the return request has been confirmed by the seller or Shopee Dispute Team, you will have 3 days to return the product. You may follow the following steps.</p>
            <h3>[Buyer Basics] What payment methods are supported by Shopee?</h3>
            <p>Shopee aims to provide all users a safe and secure platform to buy and sell online. With that, we have collaborated with various payment service provider to process such payment in a fast, convenient and secure manner. Below are 6 different payment method buyers can use to pay for their orders.</p>
            <h3>[Buyer Basics] How to cancel unpaid order and paid order before it is being shipped out?</h3>
            <p>Buyers can only request for cancellation once for every order. As such, once your request is rejected or withdrawn, you would not be able to submit another cancellation request for the same order.Once you have requested for Order cancellation, the seller is to respond to your cancellation request within 2 days (48 hours).</p>
            <h3>[Buyer Basics] How can I get the refund for my cancelled order?</h3>
            <p>Cancelled orders are orders that did not manage to be processed successfully before the item is being shipped. There are two types of cancelled orders which are cancelled before payment is being processed and cancelled after payment had been confirmed.Payments made through bank transfer will be refunded to your ShopeePay within 24 hours.</p>
        </div>
    )
}

export default Help