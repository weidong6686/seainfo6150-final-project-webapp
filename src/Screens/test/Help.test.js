import React from "react";
import Help from "../Help.jsx";

describe("Help tests", () => {
  it("renders correctly", () => {
    const { container } = (<Help />);
    expect(container).toMatchSnapshot();
  });
});