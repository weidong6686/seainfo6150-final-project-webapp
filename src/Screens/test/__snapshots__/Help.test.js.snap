// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help tests renders correctly 1`] = `undefined`;
